GVFS/GVFS.FunctionalTests/Program.cs

```diff
@@ -85,6 +85,12 @@ public static void Main(string[] args)
                 excludeCategories.Add(Categories.MacOnly);
             }
 
+            if (runner.HasCustomArg("--extra-only"))

```

Comment:
Should this check be in an `else if` after this check?

```
if (runner.HasCustomArg("--full-suite"))
```

Also, I've started adding a `Console.WriteLine` to describe what will be run (e.g. `Console.WriteLine("Running the full suite of tests");` above).

Could you do the same here?  I've found it to be a useful sanity check when looking at functional test run output.